perfkitbenchmarker/linux_benchmarks/beam_integration_benchmark.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generic benchmark running Apache Beam Integration Tests as benchmarks.
+
+This benchmark provides the piping necessary to run Apache Beam Integration
+Tests as benchmarks. It provides the minimum additional configuration necessary
+to get the benchmark going.
+"""
+
+import copy
+import datetime
+import tempfile
+
+from perfkitbenchmarker import beam_benchmark_helper
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import dpb_service
+from perfkitbenchmarker import errors
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import sample
+from perfkitbenchmarker.dpb_service import BaseDpbService
+
+BENCHMARK_NAME = 'beam_integration_benchmark'
+
+BENCHMARK_CONFIG = """
+beam_integration_benchmark:
+  description: Run word count on dataflow and dataproc
+  dpb_service:
+    service_type: dataflow
+    worker_group:
+      vm_spec:
+        GCP:
+          machine_type: n1-standard-1
+          boot_disk_size: 500
+        AWS:
+          machine_type: m3.medium
+      disk_spec:
+        GCP:
+          disk_type: nodisk
+        AWS:
+          disk_size: 500
+          disk_type: gp2
+    worker_count: 2
+"""
+
+flags.DEFINE_string(
+    'beam_it_class',
+    'org.apache.beam.examples.WordCountIT',
+    'Path to IT class')
+flags.DEFINE_string('beam_it_args', None, 'Args to provide to the IT')
+
+FLAGS = flags.FLAGS
+
+
+def GetConfig(user_config):
+  return configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
+
+
+def CheckPrerequisites(benchmark_spec):
+  """Verifies that the required resources are present.
+
+  Raises:
+    perfkitbenchmarker.data.ResourceNotFound: On missing resource.
+  """
+  if FLAGS.beam_it_args is None:
+    raise errors.Config.InvalidValue('No args provided.')
```

Comment:
The missing args error 
> InvalidValue: No args provided.
Does not provide any instructions on how to get around it.
Are you fine with this UX ?